src/Pages/ProductList/ProductList.js

```diff
@@ -1,9 +1,60 @@
 import React, { Component } from "react";
+import styled from "styled-components";
+import ProductListMain from "./ProductListMain/ProductListMain";
+import AsideMap from "./AsideMap/AsideMap";
 
 class ProductList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      headerOptionClickedAt: "",
+      pageNumClickedAt: 1,
+    };
+  }
+
+  clickHandlerBtnClicked = (name, idx) => {
+    if (this.state[`${name}ClickedAt`] === idx) return;
+
+    this.setState({
+      [`${name}ClickedAt`]: idx,
+    });
+  };
+
+  clickHandlerNextPrevBtn = (idx) => {
+    this.setState({
+      pageNumClickedAt: this.state.pageNumClickedAt + idx,
+    });
+  };
+
   render() {
-    return <div></div>;
+    const { headerOptionClickedAt, pageNumClickedAt } = this.state;
+
+    console.log(pageNumClickedAt);
+    return (
+      <div>
+        <EmptyHeaderNav />
+        <ProductListWrap>
+          <ProductListMain
+            headerOptionClickedAt={headerOptionClickedAt}
+            pageNumClickedAt={pageNumClickedAt}
+            clickHandlerBtnClicked={this.clickHandlerBtnClicked}
+            clickHandlerNextPrevBtn={this.clickHandlerNextPrevBtn}
```

Comment:
```suggestion
            gotoHandler={this.gotoHandler}
            nextPrevHandler={this.nextPrevHandler}
```
구체적인 건 좋은데 전반적으로 이름이 너무 깁니다...

Reply:
이제부터 한번 모두 줄여볼게여